launcher/FileSystem.cpp

```diff
@@ -872,6 +872,8 @@ bool createShortcut(QString destination, QString target, QStringList args, QStri
            << "\n";
     stream << "Type=Application"
            << "\n";
+    stream << "Categories=Game"
```

Comment:
we should probably use the same categories as our regular [desktop file](https://github.com/PrismLauncher/PrismLauncher/blob/36d2109a1596df5ec20cb2b1828fc25846e7d926/program_info/org.prismlauncher.PrismLauncher.desktop.in#L10)

Reply:
Changed applied.